src/utils/pgwire/src/net.rs

```diff
@@ -79,6 +80,14 @@ impl Listener {
             Self::Tcp(listener) => {
                 let (stream, addr) = listener.accept().await?;
                 stream.set_nodelay(true)?;
+                // Set TCP keepalive to 5 minutes, which is less than the connection idle timeout of 350 seconds in AWS ELB.
+                // https://docs.aws.amazon.com/elasticloadbalancing/latest/network/network-load-balancers.html#connection-idle-timeout
+                #[cfg(not(madsim))]
+                {
+                    let r = socket2::SockRef::from(&stream);
+                    let ka = socket2::TcpKeepalive::new().with_time(Duration::from_secs(300));
```

Comment:
Shall we make it a configurable value in case there's another default timeout in other environments?

Reply:
Done